lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -157,8 +160,14 @@ def self.available_options
                                        description: "Print verbose debug output",
                                        optional: true,
                                        default_value: false,
-                                       is_string: false)
+                                       is_string: false),
+          FastlaneCore::ConfigItem.new(key: :google_service_account,
```

Comment:
Rename parameter to `service_credentials_file` to keep make it consistent with the gradle plugin.